src/element.ts

```diff
@@ -169,6 +170,8 @@ export class Element {
    * Note that StaveNote calls setGroupStyle() when setStyle() is called.
    */
   addChildElement(child: Element): this {
+    if (child.parent) throw new RuntimeError('Element', 'Draw not defined');
```

Comment:
I don't understand this error message. How is it related to draw when you're checking that child already has a parent?

Reply:
Good catch. The comment is wrong.

Reply:
Error message changed